sosw/components/dynamo_db.py

```diff
@@ -561,7 +561,7 @@ def _parse_filter_expression(self, expression: str) -> Tuple[str, Dict]:
         return result_expr, result_values
 
 
-    def get_by_scan(self, attrs=None, table_name=None, strict=None, fetch_all_fields=None):
+    def get_by_scan(self, attrs=None, table_name=None, index_name=None, strict=None, fetch_all_fields=None):
```

Comment:
Since this code is already used, and we don't know if people use the argument names or not, better add the new param last

Reply:
I think that in this specific case we can inject it in the middle to allow the same order of arguments in all the functions (both get and scan). I have checked the code of both `sosw` and `tts_lambdas` - should not affect anything.